apps/app-fp-movement/src/screens/Return/ReturnListScreen.tsx

```diff
@@ -46,7 +45,6 @@ export const ReturnListScreen = () => {
     useSelector((state) => state.documents.list)?.filter((i) => i.documentType?.name === 'return') as IReturnDocument[]
   ).sort((a, b) => new Date(b.documentDate).getTime() - new Date(a.documentDate).getTime());
 
-  console.log('list', jsonFormat(list));
   const loading = useSelector((state) => state.app.loading);
 
   const [delList, setDelList] = useState<IDelList>({});
```

Comment:
The code has several issues that need attention:

1. The 'jsonFormat' import is being removed which may break functionality unless it is replaced with alternative logic.
2. The 'jsonFormat' console.log statement should be reviewed and potentially removed for production builds to avoid unnecessary logging.
3. The usage of 'useState' for 'delList' can lead to potential issues related to mutable state. Consider using immutability libraries or approaches such as the Immer library for managing immutable state updates more effectively.
4. The 'handleSendDocument' function does not handle errors or edge cases related to document sending, which can impact user experience and error handling scenarios.
5. Inconsistent variable naming conventions observed in the codebase that might affect code readability and maintenance.

Overall, the code quality rate is 5.

Quality Rate: 5/10